includes/utils/class-amp-mutation-utils.php

```diff
@@ -0,0 +1,325 @@
+<?php
+/**
+ * Class AMP_Mutation_Utils
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Mutation_Utils
+ *
+ * @since 0.7
+ */
+class AMP_Mutation_Utils {
```

Comment:
From an architectural perspective, it is interesting that all methods are `public static`. Are they really all meant to but used in that purposes?

Reply:
It would be much better to not have all of these methods `static`. This might mean bootstrapping the class somewhere else. 

It's currently calling `AMP_Validation_Utils::init();` [here in amp.php](https://github.com/Automattic/amp-wp/commit/f8aeca80dd2fe7589d6097b4c2a802ea6d4fb679#diff-4805c837d3390517773926fb26726bcdR98). It might help if we could instantiate the class somewhere, like:
```
$validation_utils = new AMP_Validation_Utils();
validation_utils->init();
```

Reply:
[amp_post_meta_box()](https://github.com/Automattic/amp-wp/blob/develop/includes/admin/functions.php#L152) might be a good model for this.

Reply:
Sure, this is not really a pressing need or a blocker though, it doesn't have to be addressed in this PR.